vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -541,162 +584,224 @@
+														<ellipse fill="rgba(222,222,222,1)"
+															class="Ellipse_8_c_Class" rx="14.5" ry="14.5" cx="14.5"
+															cy="14.5"></ellipse></svg>
+													<svg class="Ellipse_10_module_c_sidenav">
+														<ellipse fill="rgba(240,240,240,1)"
+															class="Ellipse_10_c_Class" rx="12.5" ry="12.5" cx="12.5"
+															cy="12.5"></ellipse></svg>
+													<svg class="Ellipse_9_module_c_sidenav">
+														<ellipse fill="rgba(255,255,255,1)"
+															class="Ellipse_9_c_Class" rx="10.5" ry="10.5" cx="10.5"
+															cy="10.5"></ellipse></svg>
+													<span class="icon-info Icon_awesome_info_module_m_sidenav"></span>
+												</div>[[${module.link.module.name}]]</a></li>
+										<li
+											th:if="${!#strings.isEmpty(module.link.name) AND isExhPreview == true}"><a
+											class="dropdown-item"
+											th:href="@{'/preview/'+${previewId}+'/'+${space?.id}+'/module/'+${module.link.module?.id}}"><div
+													class="Info Info_cz_module_Class">
+													<svg class="Ellipse_8_module_c_sidenav">
+														<ellipse fill="rgba(222,222,222,1)"
+															class="Ellipse_8_c_Class" rx="14.5" ry="14.5" cx="14.5"
+															cy="14.5"></ellipse></svg>
+													<svg class="Ellipse_10_module_c_sidenav">
+														<ellipse fill="rgba(240,240,240,1)"
+															class="Ellipse_10_c_Class" rx="12.5" ry="12.5" cx="12.5"
+															cy="12.5"></ellipse></svg>
+													<svg class="Ellipse_9_module_c_sidenav">
+														<ellipse fill="rgba(255,255,255,1)"
+															class="Ellipse_9_c_Class" rx="10.5" ry="10.5" cx="10.5"
+															cy="10.5"></ellipse></svg>
+													<span class="icon-info Icon_awesome_info_module_m_sidenav"></span>
+												</div>[[${module.link.name}]]</a></li>
+										<li
+											th:if="${#strings.isEmpty(module.link.name) AND isExhPreview == true}"><a
+											class="dropdown-item"
+											th:href="@{'/preview/'+${previewId}+'/'+${space?.id}+'/module/'+${module.link.module?.id}}"><div
+													class="Info Info_cz_module_Class">
+													<svg class="Ellipse_8_module_c_sidenav">
+														<ellipse fill="rgba(222,222,222,1)"
+															class="Ellipse_8_c_Class" rx="14.5" ry="14.5" cx="14.5"
+															cy="14.5"></ellipse></svg>
+													<svg class="Ellipse_10_module_c_sidenav">
+														<ellipse fill="rgba(240,240,240,1)"
+															class="Ellipse_10_c_Class" rx="12.5" ry="12.5" cx="12.5"
+															cy="12.5"></ellipse></svg>
+													<svg class="Ellipse_9_module_c_sidenav">
+														<ellipse fill="rgba(255,255,255,1)"
+															class="Ellipse_9_c_Class" rx="10.5" ry="10.5" cx="10.5"
+															cy="10.5"></ellipse></svg>
+													<span class="icon-info Icon_awesome_info_module_m_sidenav"></span>
+												</div>[[${module.link.module.name}]]</a></li>
+									</ul>
+								</div>
+							</div>
```

Comment:
all of the above code is way too much indented, it should stay formatted the way it was.

Reply:
and most of the below as well